dotnet/csharp-grpc-load/load/requests/Unary.cs

```diff
@@ -100,11 +101,31 @@ public static async Task ClientGet(uint i, SimpleCacheClient momentoClient, stri
                 await momentoClient.GetAsync(cacheName, ByteKey(i));
                 stats.getLatencyTicks = Stopwatch.GetTimestamp() - startTimestamp;
             }
-            catch (MomentoSdk.Exceptions.SdkException e)
+            catch (Momento.Sdk.Exceptions.SdkException e)
             {
                 stats.getError = e;
             }
         }
+
+        public static async Task ClientListAdd(uint i, Momento.Sdk.Incubating.SimpleCacheClient momentoClient, string cacheName, Stats stats)
+        {
+            var listName = i.ToString();
+            var value = i.ToString();
+            var startPushFrontTimestamp = Stopwatch.GetTimestamp();
+            await momentoClient.ListPushFrontAsync(cacheName, listName, value, true);
+            stats.setLatencyTicks = Stopwatch.GetTimestamp() - startPushFrontTimestamp;
+
+            var startPopBackTimestamp = Stopwatch.GetTimestamp();
+            await momentoClient.ListPopBackAsync(cacheName, listName);
+            stats.setLatencyTicks = Stopwatch.GetTimestamp() - startPopBackTimestamp;
+        }
+        
+        public static async Task ClientListFetch(uint i, Momento.Sdk.Incubating.SimpleCacheClient momentoClient, string cacheName, Stats stats)
+        {
+            var listName = i.ToString();
+            var startTimestamp = Stopwatch.GetTimestamp();
+            await momentoClient.ListFetchAsync(cacheName, listName);
+            stats.getLatencyTicks = Stopwatch.GetTimestamp() - startTimestamp;
+        }
```

Comment:
We should move this to the List.cs file as they are list operations.

Also the ClientListAdd and ClientListFetch should have the same logic as List.Add and List.fetch. We want to run the same kind of load test from the sdk client as we are running from the client protos.